examples/pgadmin/pgadmin.yaml

```diff
@@ -1,11 +1,22 @@
 apiVersion: postgres-operator.crunchydata.com/v1beta1
 kind: PGAdmin
 metadata:
-  name: rhino
+  name: pgadmin
```

Comment:
Oh, this name change was just to make my life easier while doing manual testing -- and I wasn't planning on leaving the ServerGroup in this example looking like that. I just pushed it up like this so that y'all could see what a PGAdmin CR looks like with my changes.

Making a note to change this file before the merge.